libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1007,6 +1007,16 @@ TrackListHolder WaveTrack::WideEmptyCopy(
    return result;
 }
 
+TrackListHolder WaveTrack::MonoToStereo()
+{
+   assert(NChannels() == 1);
```

Comment:
It should also be asserted, and commented with `@pre`, that `this` has null for `GetOwner()`.

Reply:
Then `//! @pre !GetOwner()` only is enough precondition as it implies that `NChannels() == 1`